configuration/examples/base/manifests/thanos-compact-statefulSet.yaml

```diff
@@ -27,6 +27,24 @@ spec:
         app.kubernetes.io/part-of: observatorium
         app.kubernetes.io/version: v0.24.0
     spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+          - podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                  - thanos-compact
+                - key: app.kubernetes.io/instance
+                  operator: In
+                  values:
+                  - observatorium-xyz
+              namespaces:
+              - observatorium
+              topologyKey: kubernetes.io/hostname
+            weight: 100
```

Comment:
Where are they from? Did you run`make manifests`?

Reply:
no, these changes come from `make update-jsonnet`.